src/components/Slider/Slider.stories.mdx

```diff
@@ -0,0 +1,503 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import Slider from "./Slider";
+import SimpleGrid from "../Grid/SimpleGrid";
+import { GridGaps } from "../Grid/GridTypes";
+import Heading from "../Heading/Heading";
+import { HeadingLevels } from "../Heading/HeadingTypes";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("Slider")}
+  component={Slider}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36889%3A25871",
+    },
+    jest: ["Slider.test.tsx"],
+  }}
+  argTypes={{
+    className: { table: { disable: true } },
+    id: { table: { disable: true } },
+    name: { table: { disable: true } },
+    onChange: { table: { disable: true } },
+  }}
+/>
+
+# Slider
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.25.4`   |
+| Latest            | `0.25.4`   |
+
+<Description of={Slider} />
+
+The text input component doubles as a display for the slider's current value.
+For this type of component, the `value` prop must be a single number.
+
+<Canvas withToolbar>
+  <Story
+    name="Slider with Props"
+    args={{
+      defaultValue: 50,
+      helperText: "Pass in a value from the min 0 to the max 100 values.",
+      invalidText: "Oh no this is an error :(",
+      isDisabled: false,
+      isInvalid: false,
+      isRangeSlider: false,
+      isRequired: false,
+      labelText: "Slider label",
+      max: 100,
+      min: 0,
+      optReqFlag: true,
+      showBoxes: true,
+      showHelperInvalidText: true,
+      showLabel: true,
+      showValues: true,
+      step: 1,
+    }}
+    argTypes={{
+      defaultValue: { table: { disable: true } },
+      isRangeSlider: { table: { disable: true } },
+    }}
+  >
+    {(args) => <Slider {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Slider with Props" />
+
+## Range Slider
+
+The text input components double as displays for the slider's current
+minimum and maximum value. For this type of component, the `value` prop must
+be an array of two numbers. This signifies the starting and ending values of
+the range slider.
+
+<Canvas withToolbar>
+  <Story
```

Comment:
With `isRangeSlider` disabled in the initial `ArgsTable`, I think it would be good to also add the `ArgsTable` for this story.  Maybe a simplified version?

Reply:
Ah yea, I'll add the same table for this story. It was tedious and Storybook crashed when I tried to switch `isRangeSlider` from false to true. This is because the `defaultValue` can be a single value or an array. So I figured it'd be easier and simpler to generate both stories.

Reply:
OK, that's what I figured.  Thanks.